index.html

```diff
@@ -53,15 +53,15 @@
 
     <!-- Title -->
 
-    <title>CALCULATOR | YOSHITHA</title>
+    <title>CALCULATOR</title>
 
 </head>
 
 
 <body>
 
 
-  <header class="text-center title">CALCULATOR | YOSHITHA</header>
+  <header class="text-center title" style="font-style: italic bold" >CALCULATOR </header>
```

Comment:
remove the italic option bcs italicizing is not an issue assigned to you @Pragya2056

Reply:
Italics was also assigned to me.

Reply:
But @JP109 was done before it @Pragya2056

Reply:
Sorry I din't realize that.

Reply:
@JP109 did this before you so I merge it so you reverse the change @Pragya2056

Reply:
This is resolved.